apps/src/lib/levelbuilder/lesson-editor/activitiesEditorRedux.js

```diff
@@ -543,6 +543,10 @@ export const mapActivityDataForEditor = rawActivities => {
       activitySection.text = activitySection.description || '';
       delete activitySection.description;
 
+      activitySection.duration = activitySection.duration || '';
```

Comment:
It looks like the default here is the empty string and below is 0. Is there a reason they're different?

Reply:
Oh good call. I will update